system/scripts/items/summoning.cs

```diff
@@ -55,23 +88,44 @@ public ItemUseResult SCR_USE_SUMMONORB_ENEMY(Character character, Item item, str
 	[On("PlayerReady")]
 	public void OnPlayerReady(object sender, PlayerEventArgs args)
 	{
-		if (!ZoneServer.Instance.Conf.World.BlueOrbFollowWarp)
-			return;
 
 		var character = args.Character;
 
-		if (!character.Variables.Perm.TryGet<DateTime>("Melia.OrbSummon.DisappearTime", out var disappearTime))
+		if (!character.Variables.Perm.TryGetInt("Melia.BlueOrbSummon.MonsterId", out var monsterClassId))
 			return;
 
-		if (DateTime.Now > disappearTime)
+		// If follow warp is not enabled, your monster is released from you when you leave a map,
+		// functionally causing the master to forget about it and be able to summon another monster.
+		if (!ZoneServer.Instance.Conf.World.BlueOrbFollowWarp) 
+		{
+			character.Variables.Perm.Remove("Melia.BlueOrbSummon.MonsterId");
```

Comment:
Again a lot of redundancy.

Reply:
This also uses the new method to clear those variables.